backend/frontend/templates/frontend/progress_details/page.html

```diff
@@ -111,6 +111,30 @@ <h6>Link to your work</h6>
                     </form>
                 </div>
             {% endif %}
+            {% if course_component.content_item.content_type_nice == "project" and course_component.content_item.project_submission_type_nice == "repo" or course_component.content_item.project_submission_type_nice == "continue_repo" %}
+                <div class="flex flex-col p-2 border-2 rounded-md gap-2">
+                    <div class="p-2 border-2 rounded-md ">
+                        <p>
+                            Repository: 
+                            <a id="repo-link" class="{{styles.link}}" href="{{course_component.repository.get_github_repo_link}}" target="_blank">{{course_component.repository.full_name}}</a>
+                        </p>
+                    </div>
+                    {% if course_component.repository.pull_requests.all|length %}
+                        <div class="p-2 border-2 rounded-md ">
+                            <p>
+                                Open pull requests: 
+                            </p>
+                            <ul class="list-disc">
+                                {% for pr in course_component.repository.pull_requests.all %}
+                                    <li class="ml-4 mb-1">
+                                        <a id="pr_{{pr.number}}" class="{{ styles.link }} underline" href="{{ course_component.repository.get_github_repo_link }}/pull/{{ pr.number }}" target="_blank">{{ pr.title }} - {{ pr.created_at }}</a>
```

Comment:
I decided to add it to the `PullRequest` model, because it seemed more appropriate because a repo may have more than one pr. Maybe I'm missing something.